### PR TITLE
Addition to subquery limit migration guide

### PR DESCRIPTION
**Pull request by writer-jill**
Addition to subquery limit migration guide to provide information on configuring Druid to emit the appropriate metrics.
Suggestion made on PR https://github.com/apache/druid/pull/16519.

This PR has:
- [x] been self-reviewed.

### CODE REVIEW

File: docs/release-info/migr-subquery-limit.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ This property takes precedence over `maxSubqueryRows`.
 You can set both `maxSubqueryRows` and `maxSubqueryBytes` at cluster level and override them in individual queries. 
 See [Overriding default query context values](../configuration#overriding-default-query-context-values) for more information.
 
+Make sure you enable `SubqueryCountStatsMonitor` so that Druid emits metrics for subquery statistics.
+To do this, add `org.apache.druid.server.metrics.SubqueryCountStatsMonitor` to the `druid.monitoring.monitors` property in your `common.runtime.properties` configuration file.
```
Comment (cryptoe) [inline docs/release-info/migr-subquery-limit.md:56]: We should mention its broker only  monitor.

Comment (writer-jill) [reply docs/release-info/migr-subquery-limit.md:56]: updated

File: docs/release-info/migr-subquery-limit.md [hunk 2]
```diff
@@ -52,6 +52,10 @@ This property takes precedence over `maxSubqueryRows`.
 You can set both `maxSubqueryRows` and `maxSubqueryBytes` at cluster level and override them in individual queries. 
 See [Overriding default query context values](../configuration#overriding-default-query-context-values) for more information.
 
+Make sure you enable the Broker monitor `SubqueryCountStatsMonitor` so that Druid emits metrics for subquery statistics.
+To do this, add `org.apache.druid.server.metrics.SubqueryCountStatsMonitor` to the `druid.monitoring.monitors` property in your `common.runtime.properties` configuration file.
```
Comment (LakshSingla) [inline docs/release-info/migr-subquery-limit.md:56]:
```suggestion
To do this, add `org.apache.druid.server.metrics.SubqueryCountStatsMonitor` to the `druid.monitoring.monitors` property in your Broker's `runtime.properties` configuration file.
```

**Review by LakshSingla — APPROVED** [review]
nit:

File: docs/release-info/migr-subquery-limit.md [hunk 3]
```diff
@@ -52,6 +52,10 @@ This property takes precedence over `maxSubqueryRows`.
 You can set both `maxSubqueryRows` and `maxSubqueryBytes` at cluster level and override them in individual queries. 
 See [Overriding default query context values](../configuration#overriding-default-query-context-values) for more information.
 
+Make sure you enable the Broker monitor `SubqueryCountStatsMonitor` so that Druid emits metrics for subquery statistics.
+To do this, add `org.apache.druid.server.metrics.SubqueryCountStatsMonitor` to the `druid.monitoring.monitors` property in your Broker's `runtime.properties` configuration file.
+See [Metrics monitors](../configuration#metrics-monitors) for more information.
```
Comment (vtlim) [inline docs/release-info/migr-subquery-limit.md:57]:
```suggestion
See [Metrics monitors](../configuration/index.md#metrics-monitors) for more information.
```

**Review by vtlim — APPROVED** [review]

**Pull request merged**